traffic_portal/test/end_to_end/common/commonFunctions.js

```diff
@@ -31,4 +31,17 @@ module.exports = function() {
 	this.urlPath = function ( url ) {
 		return '/' + String(url).split('/').slice(3).join('/');
```

Comment:
I know you didn't change this in this PR, but fwiw this kind of parsing is more reliably implemented on [the URL object](https://developer.mozilla.org/en-US/docs/Web/API/URL):

```js
>> (new URL("https://test.google.com/some/path#test-frag?param=value")).pathname
<- "/some/path"
```

Reply:
rather not touch this as part of this PR as i didn't write that